bitvision/config.ini

```diff
@@ -3,6 +3,6 @@ KEY = default-value
 SECRET = default-value
 PASSPHRASE = default-value
 
-[STATE]
+[STATUS]
```

Comment:
Not that it really matters, but I think state is more accurate here.

Reply:
That's cool, I just wanted it to not error out.

Reply:
What was the error?

Reply:
Key not found.

Reply:
Fixed.